internal/elements.go

```diff
@@ -334,6 +334,8 @@ func (t *ResourceType) Is(name xml.Name) bool {
 
 var CollectionName = xml.Name{Namespace, "collection"}
 
+var PrincipalName = xml.Name{Namespace, "principal"}
```

Comment:
Can we replace `principalName` from `elements.go` with this new shared variable?

Reply:
Got it. I pushed the refactored fix by dropping principalName
Changes was required in server.go to reference internal.PrincipalName